src/pages/Login.tsx

```diff
@@ -1,23 +1,40 @@
 import { useState, FormEvent } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
 import { LogIn } from 'lucide-react';
 import Button from '../components/Button';
 import Input from '../components/Input';
 import toast from 'react-hot-toast';
+import { authService } from '../services/api';
 
 export default function Login() {
+  const navigate = useNavigate();
   const [isLoading, setIsLoading] = useState(false);
+  const [formData, setFormData] = useState({
```

Comment:
Updated the Login page so that user is showed the new page navbar with dashboard, and Book appointments